go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/aws/aws-sdk-go-v2/config v1.18.16
-	github.com/compose-spec/compose-go v1.20.0
+	github.com/compose-spec/compose-go/v2 v2.0.0-rc.2
```

Comment:
We should wait for `compose-go/v2 v2.0.0.rc.3` release to merge this PR as we currently have regressions in this version.
There won't be any API changes just internal fixes

Reply:
Sure, let me know and I will rebase

Reply:
@crazy-max release `v2.0.0-rc.3` of `compose-go` is out, you can use it now